### PR TITLE
JAMES-1765 Add a post about James 3.0 release

### PR DESCRIPTION
**Pull request by chibenwa**
Tweet : 

```
The Apache James team is delighted to finally announce the 3.0 version of the James server.

Thanks to all contributors!

[tiny url link to the post]
```

Here is the visual of the post:

capture d ecran de 2017-06-19 10-11-56

### CODE REVIEW

File: src/homepage/_posts/2017-06-20-james-3.0.markdown [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+categories: james update
+---
+
+**Apache James version 3.0** have been released.
+
+After several years of efforts from the community, the Apache James developers are delighted to announce you the release
+of the **3.0 version** of the Apache James server.
+
+This release brings:
+ - **new protocols**: IMAP, [JMAP](https://jmap.io/) (experimental), [WebAdmin](https://james.apache.org/server/manage-webadmin.html).
```
Comment (quynhn) [inline src/homepage/_posts/2017-06-20-james-3.0.markdown]: IMAP is the new one on V3?

Comment (chibenwa) [inline src/homepage/_posts/2017-06-20-james-3.0.markdown]: Yes

Comment (aduprat) [inline src/homepage/_posts/2017-06-20-james-3.0.markdown]: Is WebAdmin a new protocol ?

**Comment by mbaechler** [comment]
We should not say Guice is a packaging thing, it's a wiring. We should probably fix the doc too.
Guice wiring uses jar packaging + manifest.

File: src/homepage/_posts/2017-06-20-james-3.0.markdown [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+ - **new back-ends**: [Cassandra](https://cassandra.apache.org/) mailbox, [ElasticSearch](https://www.elastic.co/fr/products/elasticsearch)
+ search index, [Lucene](https://lucene.apache.org/) search index amongst others.
+ - **new [packaging](https://james.apache.org/server/packaging.html)**: Especially [Guice](https://github.com/google/guice)
```
Comment (rouazana) [inline src/homepage/_posts/2017-06-20-james-3.0.markdown]: Upcase for the 3 new, to be consistent with the following points

File: src/homepage/_posts/2017-06-20-james-3.0.markdown [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+ applications, but also [Docker images](https://www.docker.com/what-docker) were contributed for common tasks and for most
+ available packaging.
+ - **Improved system administration**: [Grafana](https://james.apache.org/server/metrics.html) reporting,
+ [Kibana](https://www.elastic.co/fr/products/kibana) log collection...
+ - **Community enhancements**: New [welcoming website](https://james.apache.org/index.html), as well as documentation re-work.
+ Also we introduced a [community chat](https://gitter.im/apache/james-project). All of this aims at getting a more living
+ community and being more attractive to newcomers.
+
+Special care have been took with testing. The team works in a Test Driven Development fashion, and improved the test coverage of the project.
+We write many integration tests with [MPT](https://james.apache.org/mpt/index.html), but also using HTTP, IMAP and SMTP libraries. We
```
Comment (rouazana) [inline src/homepage/_posts/2017-06-20-james-3.0.markdown]: s/write/wrote/ ?

File: src/homepage/_posts/2017-06-20-james-3.0.markdown [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+also developed tools for load testing, based on [Gatling](https://gatling.io/): [Gatling JMAP](https://github.com/linagora/james-gatling),
+[Gatling IMAP](https://github.com/linagora/gatling-imap) and [Gatling SMTP](https://github.com/linagora/james-gatling/tree/master/src/main/scala-2.11/org/apache/james/gatling/smtp).
+
+**Apache James version 3.0** can then be used as a **Mail Delivery Agent**, but also as a **Mail Transfer Agent** allowing you to easily integrate emails with
+your applications. Read [more](https://james.apache.org/index.html#intro).
+
+If you encounter any problem with this release, please contact us [on gitter](https://gitter.im/apache/james-project) and open tickets
+on [our bug tracker](https://issues.apache.org/jira/browse/JAMES).
```
Comment (rouazana) [inline src/homepage/_posts/2017-06-20-james-3.0.markdown]: what about mentionning the good old mailing lists?

**Comment by chibenwa** [comment]
See for merging s/packaging/wiring/g fast : https://github.com/linagora/james-project/pull/851

File: src/homepage/_posts/2017-06-20-james-3.0.markdown [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+categories: james update
+---
+
+**Apache James version 3.0** have been released.
+
+After several years of efforts from the community, the Apache James developers are delighted to announce you the release
+of the **3.0 version** of the Apache James server.
+
+This release brings:
+ - **New protocols**: IMAP, [JMAP](https://jmap.io/) (experimental), .
```
Comment (strokyl) [inline src/homepage/_posts/2017-06-20-james-3.0.markdown]: we are not sure if only JMAP is experimental or both are.
I would separate IMAP and JMAP by a new line

File: src/homepage/_posts/2017-06-20-james-3.0.markdown [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+ - **New administration interface**: [WebAdmin](https://james.apache.org/server/manage-webadmin.html) is a REST administration interface.
+ - **New back-ends**: [Cassandra](https://cassandra.apache.org/) mailbox, [ElasticSearch](https://www.elastic.co/fr/products/elasticsearch)
+ search index, [Lucene](https://lucene.apache.org/) search index amongst others.
+ - [Docker images](https://www.docker.com/what-docker) were contributed for common tasks and for most available packaging.
+ - **New [wiring](https://james.apache.org/server/wiring.html)** with [Guice](https://github.com/google/guice), which ease development.
+ - **Improved system administration**: [Grafana](https://james.apache.org/server/metrics.html) reporting,
+ [Kibana](https://www.elastic.co/fr/products/kibana) log collection...
+ - **Community enhancements**: New [welcoming website](https://james.apache.org/index.html), as well as documentation re-work.
+ Also we introduced a [community chat](https://gitter.im/apache/james-project). All of this aims
+     at getting a more living
```
Comment (strokyl) [inline src/homepage/_posts/2017-06-20-james-3.0.markdown]: You can merge those two line in one

File: src/homepage/_posts/2017-06-20-james-3.0.markdown [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+Special care have been took with testing. The team works in a Test Driven Development fashion, and improved the test coverage of the project.
+We wrote many integration tests with [MPT](https://james.apache.org/mpt/index.html), but also using HTTP, IMAP and SMTP libraries. We
+also developed tools for load testing, based on [Gatling](https://gatling.io/): [Gatling JMAP](https://github.com/linagora/james-gatling),
+[Gatling IMAP](https://github.com/linagora/gatling-imap) and [Gatling SMTP](https://github.com/linagora/james-gatling/tree/master/src/main/scala-2.11/org/apache/james/gatling/smtp).
+
+**Apache James version 3.0** can then be used as a **Mail Delivery Agent**, but also as a **Mail Transfer Agent** allowing you to easily integrate emails with
+your applications. Read [more](https://james.apache.org/index.html#intro).
+
+If you encounter any problem with this release, please contact us [on gitter](https://gitter.im/apache/james-project), on the
+ [mailing lists](https://james.apache.org/mail.html) and open tickets on [our bug tracker](https://issues.apache.org/jira/browse/JAMES).
```
Comment (strokyl) [inline src/homepage/_posts/2017-06-20-james-3.0.markdown]: or open tickets on

Comment (chibenwa) [inline src/homepage/_posts/2017-06-20-james-3.0.markdown]: Why ? Why can't I do them all?

File: src/homepage/_posts/2017-06-20-james-3.0.markdown (hunk 5 above)
Comment (chibenwa) [inline src/homepage/_posts/2017-06-20-james-3.0.markdown]: Other documents states it better like the feature support matrix with version. I would not sacrifice readability for it.

Comment (chibenwa) [inline src/homepage/_posts/2017-06-20-james-3.0.markdown]: I added a (stable) after IMAP

File: src/homepage/_posts/2017-06-20-james-3.0.markdown (hunk 7 above)
Comment (strokyl) [inline src/homepage/_posts/2017-06-20-james-3.0.markdown]: or is not exclusive but and let think you have to do all of them

Comment (chibenwa) [inline src/homepage/_posts/2017-06-20-james-3.0.markdown]: Or implies imo that you have to do only one.
And it is better if the community uses all of these media, no?

Comment (strokyl) [inline src/homepage/_posts/2017-06-20-james-3.0.markdown]: I don't agree but I think there is not a universal answer to this question so let it like that

**Comment by chibenwa** [comment]
Email (links might change) for Apache internal communication team:

```
Hello,

The Apache James project [1] is pleased to announce the release of
version 3.0.0 of the Apache James Server, a mail customisable mail
server handling common email protocols.

The version 3.0.0 brings new features like the IMAP protocol, and
additional back-ends for mail storage like a Cassandra based mailbox.
We also introduced the use of the Guice wiring, enabling choosing which components
to embed by code. An HTTP interface for administration was also added.

Apache James Server can be downloaded from the following page [2].
Binary artifacts are published on Maven Central under Group Id
"org.apache.james" . Read more details on this release in the Apache 
James news for release 3.0.0 [3].

Please, remember to check the signature and fingerprint of the
downloaded files.

Have a great day,
Benoit Tellier, on behalf of tha Apache James community.

[1] james.apache.org
[2] apache.org/dist/james/server/3.0.0
[3] link to be added when publishing release
```

**Comment by chibenwa** [comment]
Team to James user ML (please review)

```
Hello,

Apache James developers are very glad to finally announce the release of Apache James Server 3.0.

After several years of efforts, it brings IMAP supports, JMAP support, additional back-end (mailbox on top of Cassandra, searching emails in ElasticSearch or Lucene), new wiring with Guice, an HTTP administration interface, a new website, and much more!

You can get more information on this release here: [link to the article].

You can download the release from here: [link to dowload].

Best regards,

[Sender]
Apache James PMC
```

**Comment by aduprat** [comment]
:+1: for all

**Comment by chibenwa** [comment]
Communication for V3 have already been done.

**Pull request closed without merging**